web/modules/custom/dpl_opening_hours/dpl_opening_hours.module

```diff
@@ -17,8 +23,7 @@ use Drupal\paragraphs\ParagraphInterface;
  * @param array &$variables
  *   Variables for the paragraph template, including the 'paragraph' entity.
  */
-function dpl_opening_hours_preprocess_paragraph__opening_hours(&$variables):
-void {
+function dpl_opening_hours_preprocess_paragraph__opening_hours(&$variables): void {
```

Comment:
This change is not related to the change at hand. I think it is nice but next time please at least keep it as a sepatate commit.